internal/policy/source/source.go

```diff
@@ -37,6 +42,7 @@ const (
 	DownloaderFuncKey key        = 0
 	PolicyKind        policyKind = "policy"
 	DataKind          policyKind = "data"
+	ConfigKind        policyKind = "config"
```

Comment:
Actually we could use either one of the other kinds and would work the same, but I wanted to add this for clarity of intent.